Graphs/AbstractGraph.js

```diff
@@ -252,13 +252,14 @@ export default class AbstractGraph extends React.Component {
     scaleColor(data, defaultColumn) {
         const {
             colors,
+            otherColors,
```

Comment:
default otherColors to {} otherwise you will get an exception if not defined.

Reply:
@natabal 
`otherColors` and `colors` is defined in `Graphs/defaultProperties.js` which is always accessible for all the graphs.